dns/zone-configs/kubernetes.io._0_base.yaml

```diff
@@ -18,6 +18,10 @@
       preference: 10
     - exchange: alt4.aspmx.l.google.com.
       preference: 10
+  - type: CAA # Which CA's can issue certs for the domain
+    values:
+    - 0 issue \"pki.goog\"
```

Comment:
Looking at https://github.com/octodns/octodns/blob/1ef140f372bc38b3523f457c966aaa92e890808b/tests/config/unit.tests.yaml it seems that what you have here is not the correct octodns syntax

Reply:
Thanks for the cheat sheet, will push an update in correct format.